packages/transaction-controller/src/TransactionController.ts

```diff
@@ -556,14 +621,117 @@ export class TransactionController extends BaseControllerV1<
 
```

Comment:
If we're also subscribing via the messenger, should we move our `onNetworkStateChange` into here?